helm_deploy/values-dev.yaml

```diff
@@ -47,6 +47,13 @@ generic-service:
       SecRuleUpdateTargetById 942150 "!ARGS:/outcomeComment/"
       SecRuleUpdateTargetById 942230 "!ARGS:/.*description/"
       SecRuleUpdateTargetById 942410 "!ARGS:/outcomeComment/"
+      # Disable rule 932110 as it only applies to Windows host servers
+      SecRuleRemoveById 932110
```

Comment:
was this rule being triggered then?

Reply:
Yeah, this started from Dom Bull getting 406 errors updating alerts.